functions/src/scripts/backfill-followers.ts

```diff
@@ -0,0 +1,47 @@
+import * as admin from 'firebase-admin'
+
+import { initAdmin } from './script-init'
+initAdmin()
+
+import { getValues } from '../utils'
+import { User } from 'common/user'
+import { Follow } from 'common/follow'
+
+const firestore = admin.firestore()
+
+async function backfillFollowers() {
+  console.log('Backfilling user follower counts')
+  const followerCounts: { [userId: string]: number } = {}
+  const users = await getValues<User>(firestore.collection('users'))
+
+  console.log(`Loaded ${users.length} users. Calculating follower counts...`)
+  for (const [idx, user] of users.entries()) {
+    console.log(`Querying user ${user.id} (${idx + 1}/${users.length})`)
+    const followsSnaps = await getValues<Follow>(
```

Comment:
I wouldn't call these snaps, because they are not firestore snapshots, they are Follow objects. Probably just `follows`

Reply:
Done, thanks for the review!